src/users/admin/admin.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiBearerAuth } from '@nestjs/swagger';
+import { Controller, Delete } from '@nestjs/common';
+
+import { Roles } from '../../roles/roles.decorator';
+import { Role } from '../../roles/role.enum';
+import { AdminService } from './admin.service';
+
+@ApiBearerAuth()
+@Roles(Role.Admin)
+@Controller('admin')
+export class AdminController {
+  constructor(private adminService: AdminService) {}
+  @Delete('reset-database')
+  async profile(): Promise<{ message: string }> {
```

Comment:
Can you rename the function to reflect what it actually does?

Reply:
Oops